build-tools/packages/build-cli/src/commands/check/changeset.ts

```diff
@@ -4,13 +4,16 @@
  */
 import { Flags } from "@oclif/core";
 import chalk from "chalk";
+import { sortPackageJson as sortJson } from "sort-package-json";
 
 import { BaseCommand } from "../../base";
 import { Repository } from "../../lib";
 
 export default class CheckChangesetCommand extends BaseCommand<typeof CheckChangesetCommand> {
 	static summary = `Checks if a changeset was added when compared against a branch. This is used in CI to enforce that changesets are present for a PR.`;
 
+	static enableJsonFlag = true;
```

Comment:
Just for clarification, what is this boolean used for?

Reply:
Good question. Answers here: https://oclif.io/docs/json